core/arch/arm/kernel/user_ta.c

```diff
@@ -313,6 +313,14 @@ static TEE_Result ta_load(const TEE_UUID *uuid,
 		goto error_return;
 	}
 
+	if (ta_head->flags & TA_FLAG_MULTI_SESSION &&
```

Comment:
@jenswi-linaro maybe not. The GP Internal Core API v1.1, table 4-11 "Trusted Application Standard Configuration Properties" says (emphasis mine): "`gpd.ta.multiSession` Boolean [...] This property is **ignored** for multi-instance Trusted Applications".
As for the keep-alive flag, the wording is a bit more ambiguous (emphasis mine again): "`gpd.ta.instanceKeepAlive` Boolean [...] This property is **meaningful only when** the `gpd.ta.singleInstance` is set to `true`"

@prime-zeng does this patch actually change the behavior of OP-TEE? In other words:
- What happens if `TA_FLAG_SINGLE_INSTANCE` is not set and `TA_FLAG_MULTI_SESSION` is set, compared to both not set?
- What happens if `TA_FLAG_SINGLE_INSTANCE` is not set and `TA_FLAG_INSTANCE_KEEP_ALIVE` is set, compared to both not set?